src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -226,7 +226,6 @@ source %{
       case Op_ExtractC:
       case Op_ExtractUB:
       // Vector API specific
-      case Op_StoreVectorScatter:
```

Comment:
It might need the partial implementation for this opcode. It's ok for me to add them in a separate patch. However, to make sure everything works well, I suggestion to add a block here, like
```
  case Op_StoreVectorScatter:
        // Currently the implementation for partial vectors are not implemented yet.
        // Will add them in a separate patch.
        return vlen * type2aelembytes(bt) == MaxVectorSize;
```

Reply:
OK. I will fix this. Thank you for your review.